components/views/login/Login.module.scss

```diff
@@ -0,0 +1,84 @@
+.content {
+    display: grid;
+    place-items: center;
+    height: calc(100vh - var(--nav-height) - var(--footer-height));
+    .formContainer {
+        height: fit-content;
+        width: 20rem;
+        background: rgba(3, 3, 3, 0.2);
+        padding: 2rem;
+        display: flex;
+        flex-direction: column;
+        .header {
+            text-align: center;
+            color: var(--off-white);
+            padding-bottom: 1rem;
+        }
+        .errorMessage {
+            color: red;
+        }
+        .formInputs {
+            display: flex;
+            flex-direction: column;
+            label {
+                display: flex;
+                flex-direction: column;
+                align-items: flex-start;
+                padding: 0.2rem 0rem;
+            }
+            input {
+                position: relative;
+                margin-bottom: 1rem;
+                width: 100%;
+                display: block;
+                font-size: 1rem;
+                color: var(--off-white);
+                padding: 8px;
+                border-radius: 6px;
+                border: none;
+                background: rgba(3, 3, 3, 0.1);
+                -webkit-transition: all 2s ease-in-out;
+                -moz-transition: all 2s ease-in-out;
+                -o-transition: all 2s ease-in-out;
```

Comment:
Don't need the 3 lines above. thats more of an older style. just the transition is good enough